### PR TITLE
Init Model

### PR DESCRIPTION
**Pull request by vuducdat**

### CODE REVIEW

File: app/models/category.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Category < ActiveRecord::Base
+  has_many :exams, dependent: :destroy
+  has_many :questions, dependent: :destroy
+
```
Comment (leminhtuan2015) [inline app/models/category.rb]: rm

File: app/models/answer.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Answer < ActiveRecord::Base
+  belongs_to :question
+
+  has_many :results
+  has_many :lesson, through: :lesson_words
+
```
Comment (leminhtuan2015) [inline app/models/answer.rb]: rm

**Comment by leminhtuan2015** [comment]
LGTM. well done .thank

File: app/models/answer.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Answer < ActiveRecord::Base
+  belongs_to :question
+
+  has_many :results
+  has_many :lesson, through: :lesson_words
```
Comment (VoTaiTri) [inline app/models/answer.rb]: what is lesson_words?

File: db/migrate/20150706031545_create_users.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :avatar
+      t.string :password_digest
+      t.string :remember_digest
+      t.integer :role
```
Comment (VoTaiTri) [inline db/migrate/20150706031545_create_users.rb]: should string type

**Pull request merged**